site/src/templates/componentPage.tsx

```diff
@@ -83,7 +83,7 @@ export default ({ data, pageContext, location }) => {
         </Sidebar>
         <Content>
           <ContentNeedToKnowSection listOfTips={md.frontmatter.needToKnow} />
-          {renderStorybookIFrame()}
+          {md.frontmatter.title === "Overview" ? null : renderStorybookIFrame()}
```

Comment:
```suggestion
          {md.frontmatter.title === "Overview" && renderStorybookIFrame()}
```
I find this easier to read, but no biggie as both do the job.

Reply:
The commit suggestion feature is pretty cool - I've never used that before. I like Ruby's `unless` and Elm's `if ... then ... else`, but yeah I think it's more idiomatic JS.

Reply:
I just find it weird that boolean logic is used for control flow - seems to be just taking advantage of the fact that right side of the boolean operator happens to not get executed if the left one fails.

Reply:
I agree, it is weird lol. It seems to be pretty common in React – I guess because people see the `else` bit as unnecessary.